test_programs/plonky2_prove_success/factorization/src/factor.nr

```diff
@@ -0,0 +1,89 @@
+// factor.nr
+pub fn print_factors(mut number: u32) {
+    // the number of primes limits the input to (59*59)^2 - 1 = 12117360
+    // the maximum factor limits the input to 256*256 - 1 = 65535
+    assert(number < 65536);
+    let mut factor_is_first = true;
+    //if number < 0 {
+    //    number = number * -1;
+    //    println(-1);
+    //    factor_is_first = false;
+    //}
```

Comment:
Please, remove dead code. The function `print_factors` takes only unsigned integers.

Reply:
Done: https://github.com/blocksense-network/noir/pull/56/commits/18eb0e38a720945170e86542cd8b04c22dcb7a9f